.github/workflows/cd.yaml

```diff
@@ -106,20 +106,26 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - run: |
           set -x
+          aws eks --region us-east-2 update-kubeconfig --name zq-test
+          kubectl create ns ci-master || true
+          kubectl config set-context --current --namespace=ci-master
           export ZQD_AURORA_USER=cimaster
+          export ZQD_AURORA_PW=$(kubectl get secret aurora --template="{{ index .data \"postgresql-password\" }}" | base64 --decode)
```

Comment:
```suggestion
          export ZQD_AURORA_PW=$(kubectl get secret aurora --template='{{ index .data "postgresql-password" }}' | base64 --decode)
```

Reply:
Single quotes in bash do not interpolate, and are not interchangeable with double quotes. Even if single-quotes work in this line of bash, they would be confusing since the expression wrapped by {{ }} looks like a form of interpolation that is specific to GitHub yaml.